src/main/java/com/intershop/tool/architecture/report/common/resources/URILoader.java

```diff
@@ -35,22 +42,23 @@ else if (uri.getScheme().startsWith("file"))
         }
     }
 
-    private static InputStream getInputStream(WebTarget webTarget) throws IOException
+    private static InputStream getInputStream(HttpRequest request) throws IOException, InterruptedException
     {
-        Response response = webTarget.request().get();
-        if (Status.OK.getStatusCode() == response.getStatus())
+        HttpClient client = HttpClient.newHttpClient();
+        HttpResponse<String> response = client.send(request, BodyHandlers.ofString());
```

Comment:
You could use `HttpResponse.BodyHandlers.ofInputStream()` directly.

Reply:
Good catch - is refactored now